docs/89-NEWS.md

```diff
@@ -62,6 +62,14 @@ This document was last updated at commit v4.3-197-g9085ed631.
   objects. If you used these low level APIs to add keys and buttons dynamically,
   please migrate your code to the corresponding `:append_` and `:remove_`
   client methods.
+ * `beautiful.border_width` and `beautiful.border_color` are not honored even
```

Comment:
i see `border_width` is still set in the updated themes above (or they're not finished yet?)

also i think it makes sense to leave smth like `beautiful.client_border_width` to be able to set default width for all the clients disregard their state (while not affecting other widgets which have border)

and the same for `client_border_color`, as i could easily imagine the scenario when somebody would like to set one color as default one, and set explicitly colors only for some of the states

Reply:
s/not/now/

Sorry for the "typo", but good catch. So half of what you asked was already there, but I did add a `beautiful.border_color` field as a generic fallback to address the other half. I also added a `fullscreen` variant to address another issue (well, kind of, the bug is still there but it will be easier to fix).